api/app/Console/Commands/SendNotificationsSystem.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Notifications\System as SystemNotification;
+use Error;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\View;
+use Throwable;
+
+class SendNotificationsSystem extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send-notifications:system
+                            {viewGroup : The group of views to render the notification with}
+                            {emailAddress? : The email address of the user to send to}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send system notifications from blade templates';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $successCount = 0;
+        $failureCount = 0;
+
+        $viewGroup = $this->argument('viewGroup');
+
+        // find the views in api/resources/views/
+        $emailSubjectEn = $viewGroup.'.email_subject_en';
+        $emailSubjectFr = $viewGroup.'.email_subject_fr';
+        $emailContentEn = $viewGroup.'.email_content_en';
+        $emailContentFr = $viewGroup.'.email_content_fr';
+        $inAppMessageEn = $viewGroup.'.in_app_message_en';
+        $inAppMessageFr = $viewGroup.'.in_app_message_fr';
+        $inAppHrefEn = $viewGroup.'.in_app_href_en';
+        $inAppHrefFr = $viewGroup.'.in_app_href_fr';
+
+        collect([
+            $emailSubjectEn,
+            $emailSubjectFr,
+            $emailContentEn,
+            $emailContentFr,
+            $inAppMessageEn,
+            $inAppMessageFr,
+            $inAppHrefEn,
+            $inAppHrefFr,
+        ])->each(function ($viewName) {
+            if (! View::exists($viewName)) {
+                throw new Error('View not found: '.$viewName);
+            }
+        });
+
+        $singleEmailAddress = $this->argument('emailAddress');
+        if (! is_null($singleEmailAddress)) {
+            // single email address provided
+            $recipientUsers = [
+                User::where('email', $singleEmailAddress)->sole(),
```

Comment:
is this for testing purpose?

Reply:
Yeah, exactly.  You can try sending yourself the email first, before sending it to every single user.